src/libraries/scroller.js

```diff
@@ -697,7 +697,7 @@ define(['browser', 'layoutManager', 'dom', 'focusManager', 'ResizeObserver', 'sc
 
         function onFrameClick(e) {
             if (e.which === 1) {
-                var focusableParent = focusManager.focusableParent(e.target);
+                var focusableParent = focusManager.default.focusableParent(e.target);
```

Comment:
In files such as `src\controllers\hometab.js`, where it has not been converted to an ES6 module yet, you need to add the following to the start of the file:
`    focusManager = focusManager.default || focusManager;` or add the `.default` to focusManager as you have done here.

Either works, but the first one saves time by reducing the number of lines that need to be changed if there is a lot of references of focusManager

To test this, go into tv mode in the display user settings.

Reply:
You seem to be missing a few changes to older modules.

Modules that need `focusManager = focusManager.default || focusManager;` are:

* tunerPicker.js
* filtermenu.js
* guide.js
* slideshow.js
* sortmenu.js
* subtitleeditor.js
* viewManager.js
* list.js
* scroller.js
* scrollHelper.js
* serverNotifications.js

Otherwise, this look good :)